src/protocols/bdx/tests/TestBdxTransferSession.cpp

```diff
@@ -112,34 +114,32 @@ void VerifyBdxMessageType(nlTestSuite * inSuite, void * inContext, const System:
 // Helper method for verifying that a PacketBufferHandle contains a valid StatusReport message and contains a specific StatusCode.
 void VerifyStatusReport(nlTestSuite * inSuite, void * inContext, const System::PacketBufferHandle & msg, StatusCode code)
 {
-    CHIP_ERROR err      = CHIP_NO_ERROR;
-    uint16_t headerSize = 0;
+    CHIP_ERROR err = CHIP_NO_ERROR;
     PayloadHeader payloadHeader;
-    uint16_t generalCode = 0;
-    uint32_t protocolId  = 0;
-    BitFlags<StatusCode> protocolCode;
 
     if (msg.IsNull())
     {
         NL_TEST_ASSERT(inSuite, false);
         return;
     }
 
-    err = payloadHeader.Decode(msg->Start(), msg->DataLength(), &headerSize);
-    NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
-    NL_TEST_ASSERT(inSuite, payloadHeader.HasMessageType(Protocols::SecureChannel::MsgType::StatusReport));
-    if (headerSize > msg->DataLength())
+    System::PacketBufferHandle msgCopy = System::PacketBufferHandle::NewWithData(msg->Start(), msg->DataLength());
```

Comment:
```suggestion
    System::PacketBufferHandle msgCopy = msg.CloneData();
```

Reply:
just added this in the [commit here](https://github.com/project-chip/connectedhomeip/pull/5359/commits/f0b20560ab8af4f21d1487840ebd566bc0b0807f)